### DISCUSSION TITLE
Json Error

### DISCUSSION THREAD

**Discussion by mrtozkn34** [theosanct0s/ECC-HaxBall-BOT#5] [Q&A] [answered]
Why am I getting an error when I try to use your bot?
Uncaught SyntaxError: Unexpected number in JSON at position 1
    at JSON.parse (<anonymous>)
    at <anonymous>:753:162
    at Array.forEach (<anonymous>)
    at <anonymous>:753:30

Also, what should I do to become an admin?

**Comment by theosanct0s** [comment, marked as answer]
A new version has been released, with the bugs fixed! I hope you enjoy. <3
